.github/workflows/_ci.yaml

```diff
@@ -689,3 +690,35 @@ jobs:
     with:
       MAXTEXT_IMAGE: ${{ needs.build-maxtext.outputs.DOCKER_TAG_FINAL }}
     secrets: inherit
+  
+  test-jetstream:
+    needs: build-maxtext
+    if: inputs.ARCHITECTURE == 'amd64' # no arm64 gpu runners
+    uses: ./.github/workflows/_test_unit.yaml
+    with:
+      TEST_NAME: jetstream
+      EXECUTE: |
+        docker run --shm-size=1g --gpus all ${{ needs.build-maxtext.outputs.DOCKER_TAG_FINAL }} \
+        bash <<"EOF" |& tee test-jetstream.log
+          cd /opt/jetstream
+          pip install -r requirements.txt
+          export CUDA_VISIBLE_DEVICES=0
```

Comment:
Can we test multi-GPU functionality?

Reply:
I did not find JetStream unit-test examples on GitHub for multi-GPU. Even more: afaik, it does not support GPUs so far. But it will start GPU support in the nearest future.